Automattic/jetpack
# CLI: Add Changelog Validate

Adds ability to use changelogger's validate command in Jetpack CLI.

#### Changes proposed in this Pull Request:

* Adds ability to use changelogger's validate command in Jetpack CLI.

#### Jetpack product discussion


p9dueE-2A1-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Checkout the branch
* Try `jetpack changelog validate` to have it validate a project.
* Try passing through a specific project, i.e. `jetpack changelog validate plugins/jetpack`
* You can use some passthroughs as well, like:
	* --no-strict - will still pass as long as the only issues are warnings and not errors
	* --base-dir [path] will let you set the relative path of the project so the return isn't so verbose, ex `jetpack changelog validate plugins/jetpack --base-dir /Users/you/jetpack-dev/jetpack`
	* --gh-action - returns the errors formatted as a github action file. 
* Break a changelog in a project, like by removing the significance altogether.
* Run the validator for that project and make sure it displays the error messages.
* Break it some other ways.
